content/en/account_management/rbac/permissions.md

```diff
@@ -69,6 +69,7 @@ Find below the list of permissions for the log configuration assets and log data
 | logs_write_pipelines         | Update a subset of the log pipelines       | true     |
 | logs_write_processors        | Update the log processors in an index      | true     |
 | logs_write_archives          | Update the external archives configuration | false    |
+| logs_read_archives.          | Rehydrate logs from that archive. See archive configuration details | true    |
```

Comment:
```suggestion
| logs_read_archives          | Rehydrate logs from that archive; see archive configuration details | true    |
```

Reply:
Please, don't wait for my approval for such edits. Just do it :)